BatteryMonitoring.Tests/BatteryCurrentMeasurementsTests.cs

```diff
@@ -0,0 +1,49 @@
+using Xunit;
+using System.Collections.Generic;
+namespace BatteryMonitoring.Tests
+{
+    public class BatteryCurrentMeasurementsTests
+    {
+        BatteryCurrentMeasurements currentMonitoring = new BatteryCurrentMeasurements();
+        List<int> currentReadingsList = new List<int>() { 3, 3, 5, 4, 10, 11, 12 };
+        [Fact]
+        public void GivenEmptyList_WhenListIsEmpty_TrueIsReturned()
+        {
+            List<int> emptyReadingsList = new List<int>();
+            bool actualResult = currentMonitoring.IsReadingsListEmpty(emptyReadingsList);
+            Assert.True(actualResult);
+        }
+        [Fact]
+        public void GivenReadingsList_WhenListHaveValues_FalseIsReturned()
+        {
+            List<int> readingsListWithValues = new List<int>() { 3, 3, 5, 4, 10, 11, 12 };

```

Comment:
readingsListWithValues  same as currentReadingsList

Reply:
The testcase is to check whether the given list has values or not.